tools/apiview/parsers/cpp-api-parser/ParseTests/TestCases/ExpressionTests.cpp

```diff
@@ -14,10 +14,12 @@ struct HelperClass
 class TestClass {
 public:
   std::string PublicField1;
+  [[deprecated]]
```

Comment:
We own this tool completely.  Can we simply remove?

Reply:
This is from the test case for the [[deprecated]] feature logic.